service/operator/service.go

```diff
@@ -27,37 +30,35 @@ type Config struct {
 // DefaultConfig provides a default configuration to create a new service by
 // best effort.
 func DefaultConfig() Config {
-	var err error
-
-	var k8sClient kubernetes.Interface
-	{
-		config := k8s.DefaultConfig()
-		k8sClient, err = k8s.NewClient(config)
-		if err != nil {
-			panic(err)
-		}
-	}
-
-	var newLogger micrologger.Logger
-	{
-		config := micrologger.DefaultConfig()
-		newLogger, err = micrologger.New(config)
-		if err != nil {
-			panic(err)
-		}
-	}
```

Comment:
I actually wanted to do this separately but I couldn't properly test the operator is starting properly locally with this. So I just got rid of the default configuration. This decision was made some time ago. Just boyscouting here.